cmd/node/main.go

```diff
@@ -1350,9 +1368,11 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	err = networkComponents.NetMessenger.Close()
 	log.LogIfError(err)
 
-	handleAppClose(log, sig)
-
-	return nil
+	select {
```

Comment:
This doesn't work as expected as this point is never reached (the process will wait to execute L1368)
Please remove the new code, just replace L1368-L1369 with:
```GO
go func(){
    err = networkComponents.NetMessenger.Close()
    log.LogIfError(err)
}()

time.Sleep(time.Second) //or a const here
```

Reply:
Ok. Will do this.